src/components/Tools/AdvisorCard.vue

```diff
@@ -31,8 +31,8 @@
 import { defineComponent } from 'vue';
 import store from '@/store';
 import { getCollegeFullName, getMajorFullName } from '@/utilities';
-import { AdvisorPackage } from '@/requirements/tools-types';
-import getAdvisor from '@/requirements/tools-utilities';
+import { AdvisorPackage } from '@/tools/types';
+import getAdvisor from '@/tools/tools-utilities';
```

Comment:
Can we change `tools/tools-utilities` to `tools/utilities`?

Reply:
Honestly they are more like `advisor-utilities`, so I'll change it to that.